src/shared/mkfs-util.c

```diff
@@ -235,12 +235,13 @@ int make_filesystem(
                                       "-U", vol_id,
                                       node, NULL);
 
-                else if (streq(fstype, "mksquashfs"))
+                else if (streq(fstype, "squashfs"))
```

Comment:
did you mean to squash this back in the previous commit?

Reply:
oops